### PR TITLE
Feature / Deals

### PR DESCRIPTION
**Pull request by aclarembeau**
Make sure you've checked off all these things before submitting:

- [ ] This pull request isn't for a company's fork, it's intended for the upstream Slate shared by everybody.
- [ ] This pull request is submitted to the `dev` branch.
- [ ] If it makes frontend changes, this pull request has been tested in the latest version of Firefox, Chrome, IE, and Safari.

### CODE REVIEW

File: source/includes_main/_deals.md [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+
+### Parameters
+
+Attribute | Filterable? | Description
+--------- | ---------- | -----------
+id | *NULL* | A unique identifier for the deal
+stage_id<br />**required** - *integer* | *NULL* | The stage of the deal
+title | *NULL* | The name of the deal
+value | *NULL* | The value of the deal
+currency_code | *NULL* | The code of the currency used for this deal (EUR, USD, …)
```
Comment (Guaker13) [inline source/includes_main/_deals.md]: At this point the currency_code can not be used in post/patch requests, it has to stay the one of the pipeline.

File: source/includes_main/_deals.md [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+
+Updates the specified deal by setting the values of the parameters passed. Any parameters not provided will be left unchanged.
+
+### Parameters
+Parameter | Description
+--------- | -----------
+id<br />**required** - *integer* | A unique identifier for the deal
+title | The name of the deal
+value | The value of the deal
+currency_code | The code of the currency used for this deal (EUR, USD, …)
```
Comment (Guaker13) [inline source/includes_main/_deals.md]: Same as above.

File: source/includes_main/_pipelines.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+```
+
+Updates the specified pipeline by setting the values of the parameters passed. Any parameters not provided will be left unchanged.
+
+### Parameters
+Parameter | Description
+--------- | -----------
+id<br />**required** - *integer* | The ID of the pipeline to update
+name<br />*string* | The pipeline's name
+currency_code<br />*string* | The pipeline's currency code
```
Comment (Guaker13) [inline source/includes_main/_pipelines.md:108]: Shouldn't we add the `stages_order`?

File: source/index.html.md [hunk 4]
```diff
@@ -18,9 +18,12 @@ includes:
 
 includes_main:
   - account
```
Comment (Guaker13) [inline source/index.html.md]: Let's re-organize pages in a more logical way, I suggest the following:

- prospects
- organizations
- pipelines
- stages
- deals
- automations
- automation_steps
- enrollments
- account
- users
- lists
- custom_fields
- exclusion_list_items

**Review by Guaker13 — CHANGES_REQUESTED** [review]
It looks complete, I suggest the following changes.

File: source/includes_main/_pipelines.md (hunk 3 above)
Comment (aclarembeau) [reply source/includes_main/_pipelines.md:108]: We already have position in stage. Isn't it enough?

Comment (Guaker13) [reply source/includes_main/_pipelines.md:108]: I don't see the `position` attr in the doc, you might have forgotten to mention it.

**Pull request merged**